sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/models/ShareFileHttpHeaders.java

```diff
@@ -13,9 +13,9 @@
  * Additional parameters for a set of operations, such as: Files_create,
  * Files_setHTTPHeaders.
  */
-@JacksonXmlRootElement(localName = "file-http-headers")
+@JacksonXmlRootElement(localName = "share-file-http-headers")
 @Fluent
-public final class FileHttpHeaders {
+public final class ShareFileHttpHeaders {
```

Comment:
I see this class showing up twice, once in `implementation.models` and here in `models`, was this one changed by hand or is the other a generation artifact from previous generations that had issues?

Reply:
Searched around the code. It is deleted